ui/manifests/ingress.yaml

```
@@ -6,6 +6,7 @@ metadata:
   labels:
     application: "postgres-operator-ui"
 spec:
+  ingressClassName: "ingress-nginx"
```

```suggestion
  # ingressClassName: "ingress-nginx"
```
better we only comment it